src/ui/public/sortable_column/sortable_column.js

```diff
@@ -45,8 +45,17 @@ app.directive('sortableColumn', function () {
       }
 
       getAriaLabel() {
-        const direction = this.isSortedAscending() ? 'descending' : 'ascending';
-        return `Sort ${this.field} ${direction}`;
+        const isAscending = this.isSortedAscending();
+        if(isAscending) {
+          return i18n('common.ui.sortableColumn.sortAscendingAriaLabel', {
+            defaultMessage: 'Sort {field} ascending',
+            values: { field: this.field },
+          });
+        }
+        return i18n('common.ui.sortableColumn.sortDescendingAriaLabel', {
+          defaultMessage: 'Sort {field} descending',
+          values: { field: this.field },
+        });
```

Comment:
Could keep the original string interpolation logic to avoid duplicating the object passed into `i18n`?

```
const direction = this.isSortedAscending() ? 'descending' : 'ascending';
const defaultMessage = `Sort ${this.field} ${direction}`;
return i18n('common.ui.sortableColumn.sortAscendingAriaLabel', {
            defaultMessage,
            values: { field: this.field },
          });
```

Reply:
Sorry, new to this codebase, realizing that this doesn't really work!